client/src/components/Draftail/index.js

```diff
@@ -175,9 +175,9 @@ const initEditor = (selector, originalOptions, currentScript) => {
 
   const styles = getComputedStyle(document.documentElement);
   const colors = {
-    standardHighlight: styles.getPropertyValue('--w-color-secondary-100'),
-    overlappingHighlight: styles.getPropertyValue('--w-color-secondary-50'),
-    focusedHighlight: styles.getPropertyValue('--w-color-secondary'),
+    standardHighlight: styles.getPropertyValue('--w-color-secondary-75'),
+    overlappingHighlight: styles.getPropertyValue('--w-color-warning-50'),
+    focusedHighlight: styles.getPropertyValue('--w-color-secondary-75'),
```

Comment:
I've left this duplicated in here for now in case we want to change this value in the future.

Reply:
I've made the focusedHighlight color to be darker now `secondary-100`. I found this still passes a AA contrast ratio (tested in chrome)
<img width="153" alt="Screen Shot 2022-08-25 at 3 53 49 PM" src="https://user-images.githubusercontent.com/25041665/186776032-56c849ed-2539-4b80-b1cb-5cd5aa4016e8.png">